docs/composition.ipynb

```diff
@@ -0,0 +1,739 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (40, 5)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>YEAR</th><th>QUARTER</th><th>ILOSTAT</th><th>Mean Hours Worked</th><th>Median Hours Worked</th></tr><tr><td>i64</td><td>i64</td><td>i64</td><td>f64</td><td>i64</td></tr></thead><tbody><tr><td>2004</td><td>1</td><td>1</td><td>37.231225</td><td>25</td></tr><tr><td>2004</td><td>2</td><td>1</td><td>41.136433</td><td>55</td></tr><tr><td>2004</td><td>3</td><td>1</td><td>37.531945</td><td>55</td></tr><tr><td>2004</td><td>4</td><td>1</td><td>37.375663</td><td>55</td></tr><tr><td>2005</td><td>1</td><td>1</td><td>35.841911</td><td>55</td></tr><tr><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td></tr><tr><td>2012</td><td>4</td><td>1</td><td>38.014647</td><td>25</td></tr><tr><td>2013</td><td>1</td><td>1</td><td>37.630491</td><td>25</td></tr><tr><td>2013</td><td>2</td><td>1</td><td>35.786751</td><td>55</td></tr><tr><td>2013</td><td>3</td><td>1</td><td>38.219257</td><td>55</td></tr><tr><td>2013</td><td>4</td><td>1</td><td>37.850275</td><td>25</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (40, 5)\n",
+       "┌──────┬─────────┬─────────┬───────────────────┬─────────────────────┐\n",
+       "│ YEAR ┆ QUARTER ┆ ILOSTAT ┆ Mean Hours Worked ┆ Median Hours Worked │\n",
+       "│ ---  ┆ ---     ┆ ---     ┆ ---               ┆ ---                 │\n",
+       "│ i64  ┆ i64     ┆ i64     ┆ f64               ┆ i64                 │\n",
+       "╞══════╪═════════╪═════════╪═══════════════════╪═════════════════════╡\n",
+       "│ 2004 ┆ 1       ┆ 1       ┆ 37.231225         ┆ 25                  │\n",
+       "│ 2004 ┆ 2       ┆ 1       ┆ 41.136433         ┆ 55                  │\n",
+       "│ 2004 ┆ 3       ┆ 1       ┆ 37.531945         ┆ 55                  │\n",
+       "│ 2004 ┆ 4       ┆ 1       ┆ 37.375663         ┆ 55                  │\n",
+       "│ 2005 ┆ 1       ┆ 1       ┆ 35.841911         ┆ 55                  │\n",
+       "│ …    ┆ …       ┆ …       ┆ …                 ┆ …                   │\n",
+       "│ 2012 ┆ 4       ┆ 1       ┆ 38.014647         ┆ 25                  │\n",
+       "│ 2013 ┆ 1       ┆ 1       ┆ 37.630491         ┆ 25                  │\n",
+       "│ 2013 ┆ 2       ┆ 1       ┆ 35.786751         ┆ 55                  │\n",
+       "│ 2013 ┆ 3       ┆ 1       ┆ 38.219257         ┆ 55                  │\n",
+       "│ 2013 ┆ 4       ┆ 1       ┆ 37.850275         ┆ 25                  │\n",
```

Comment:
Any chance you have a sign problem? Instead of picking the median, it seems to be picking one of the extremes.